docs/packages.md

```diff
@@ -179,56 +173,12 @@ To perform _all_ of the steps at once - build and push out the image for whateve
+linuxkit pkg push «path-to-package»
 ```
 
 #### Prerequisites
 
 * For all of the steps, you *must* be logged into hub (`docker login`).
-* For the manifest steps, you must be logged into hub and the passphrase for the key *must* be supplied as an environment variable. The build process has to resort to using `expect` to drive `notary` so none of the credentials can be entered interactively.
-
-Since it is not very good to have your passphrase in the clear (or
-even stashed in your shell history), we recommend using a password
-manager with a CLI interface, such as LastPass or `pass`. You can then
-invoke the build like this (for LastPass):
-
-```
-DOCKER_CONTENT_TRUST_REPOSITORY_PASSPHRASE=$(lpass show <key> --password) linuxkit pkg push «path-to-package»
-```
-or alternatively you may add the command to `~/.moby/linuxkit/config.yml` e.g.:
-```
-pkg:
-  content-trust-passphrase-command: "lpass show <key> --password"
-```
-
-#### Signing Manually
-
-If, for whatever reason, you want to sign an individual tag manually, whether the index (a.k.a. "multi-arch manifest") or the architecture-specific manifest, do the following:
-
-1. Make sure you have ready your credentials:
-   * docker hub login and passphrase
-   * docker notary signing key passphrase
-1. Get the following information:
-   * the name of the image repository you want to sign, including the registry host but **not** including the tag, e.g. `linuxkit/containerd`
-   * the tag of the image you want to sign, e.g. `a4aa19c608556f7d786852557c36136255220c1f` or `v5.0`
-   * the size of the image you want to sign in bytes, e.g. `1052`. See below for information on how to get this.
-   * the hash of the manifest or index to which the tag points, **not** including the `sha256:` leader, e.g. `66b3d74aeb855f393ddb85e7371a00d5f7994cc26b425825df2ce910583d74dc`. See below for information on how to get this.
-1. Set env vars with the following:
-   * `IMAGE`: name of the image, e.g. `IMAGE=docker.io/linuxkit/containerd`
-   * `TAG`: the tag you want to sign. It could be a tag pointing at a multi-arch manifest or tag pointing at an individual architecture's manifest, e.g. `TAG=a4aa19c608556f7d786852557c36136255220c1f` or `TAG=a4aa19c608556f7d786852557c36136255220c1f-s390x`
-   * `SIZE`: size of the pointed-at manifest or index, e.g. `SIZE=1052`
-   * `HASH`: sha256 hash of the pointed-at manifest or index, e.g. `HASH=66b3d74aeb855f393ddb85e7371a00d5f7994cc26b425825df2ce910583d74dc`
-1. Run the command: `notary -s https://notary.docker.io -d ~/.docker/trust addhash -p $IMAGE $TAG $SIZE --sha256 $HASH  -r targets/releases`
-
-For example:
-
-```console
-IMAGE=docker.io/linuxkit/containerd
-TAG=a4aa19c608556f7d786852557c36136255220c1f
-SIZE=1052
-HASH=66b3d74aeb855f393ddb85e7371a00d5f7994cc26b425825df2ce910583d74dc
-notary -s https://notary.docker.io -d ~/.docker/trust addhash -p $IMAGE $TAG $SIZE --sha256 $HASH  -r targets/releases
-```
 
 ##### Getting Size and Hash
```

Comment:
IIRC the size was only needed for signing. so can probably remove this section as well

Reply:
Done